apps/studio/prisma/scripts/migrations/addOrdering/addOrdering.ts

```diff
@@ -0,0 +1,89 @@
+import { DEFAULT_CHILDREN_PAGES_BLOCK } from "@opengovsg/isomer-components"
+
+import { db, sql } from "~/server/modules/database"
+import { updateBlobById } from "~/server/modules/resource/resource.service"
+
+export const up = async () => {
+  // Step 1: select all index pages and their blobs where
+  // the text `"type": "childrenpages"` isn't found
+  const sqlQuery = sql<{
+    id: string
+    siteId: string
+    content: PrismaJson.BlobJsonContent
+  }>`
+    SELECT
+      *
+    FROM
+      (
+        SELECT
+          COALESCE("db"."content", "pb"."content") AS "content",
+          "Resource"."id", 
+          "Resource"."siteId"
+        FROM
+          "Resource"
+          LEFT JOIN "Version" ON "Version"."id" = "Resource"."publishedVersionId"
+          LEFT JOIN "Blob" AS "pb" ON "pb"."id" = "Version"."blobId"
+          LEFT JOIN "Blob" AS "db" ON "db"."id" = "Resource"."draftBlobId"
+        WHERE
+          "type" = 'IndexPage'
+      ) AS "indexContent"
+    WHERE
+      ("content"->'content')::text ILIKE '%"type": "childrenpages"%' and "content" ->> 'layout' = 'index' and ("content" -> 'content')::text NOT ILIKE '%"childrenPagesOrdering"%';
```

Comment:
question(non-blocking): to read back this condition, it is looking for:
- Layout is `index`, and
- Page contains `childrenpages` block, and
- Page does not contain `childrenPagesOrdering`

Is this interpretation correct? Cos seems a bit different from the PR description that says we are adding the `childrenpages` block to all index pages if it doesn't exist.